### PR TITLE
HydrologicEnginerringCenter.HEC-RAS version 6.1.0.0

### PR DESCRIPTION
**Pull request by TechnicaVivunt**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/39314)

### CODE REVIEW

File: manifests/h/HydrologicEnginerringCenter/HEC-RAS/6.1.0.0/HydrologicEnginerringCenter.HEC-RAS.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: HydrologicEnginerringCenter.HEC-RAS
+PackageVersion: 6.1.0.0
+InstallerSwitches:
+  Silent: /S /v/qn
+Installers:
+- Architecture: x86
```
Comment (Trenly) [inline manifests/h/HydrologicEnginerringCenter/HEC-RAS/6.1.0.0/HydrologicEnginerringCenter.HEC-RAS.installer.yaml:9]: I'm not sure why it's not installing silently. Maybe try using x64 architecture?
```suggestion
- Architecture: x64
```

File: manifests/h/HydrologicEnginerringCenter/HEC-RAS/6.1.0.0/HydrologicEnginerringCenter.HEC-RAS.yaml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: HydrologicEnginerringCenter.HEC-RAS
```
Comment (ItzLevvie) [inline manifests/h/HydrologicEnginerringCenter/HEC-RAS/6.1.0.0/HydrologicEnginerringCenter.HEC-RAS.yaml:4]:
```suggestion
PackageIdentifier: HydrologicEngineeringCenter.HEC-RAS
```

File: manifests/h/HydrologicEnginerringCenter/HEC-RAS/6.1.0.0/HydrologicEnginerringCenter.HEC-RAS.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: HydrologicEnginerringCenter.HEC-RAS
```
Comment (ItzLevvie) [inline manifests/h/HydrologicEnginerringCenter/HEC-RAS/6.1.0.0/HydrologicEnginerringCenter.HEC-RAS.locale.en-US.yaml:4]:
```suggestion
PackageIdentifier: HydrologicEngineeringCenter.HEC-RAS
```

File: manifests/h/HydrologicEnginerringCenter/HEC-RAS/6.1.0.0/HydrologicEnginerringCenter.HEC-RAS.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: HydrologicEnginerringCenter.HEC-RAS
```
Comment (ItzLevvie) [inline manifests/h/HydrologicEnginerringCenter/HEC-RAS/6.1.0.0/HydrologicEnginerringCenter.HEC-RAS.installer.yaml:4]:
```suggestion
PackageIdentifier: HydrologicEngineeringCenter.HEC-RAS
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Pull request closed without merging**